--- master.mako	2010-09-06 18:06:08.643612737 -0600
+++ master.mako-mod	2010-09-06 18:05:21.233660950 -0600
@@ -171,6 +171,8 @@
     <tr>
         <td>
             <div id="footer">
+                <br /><a href="http://clearcorp.co.cr" target="_blank"><img src="/static/images/powered_by_clearcorp.png" border="0"/></a><br /><br />
+${_("Running Server:")} <span>${rpc.session.protocol}://${rpc.session.host}:${rpc.session.port} - database: ${rpc.session.db or 'N/A'}</span><br/>
 ${_("Copyright &copy; 2007-TODAY Tiny ERP Pvt. Ltd. All Rights Reserved.")|n}
 ${_("More Information on %s") % ("""<a href="http://openerp.com">http://openerp.com</a>.""")|n}
 <br/>
@@ -178,7 +180,6 @@
     tiny="""Tiny (<a href="http://tiny.be">http://tiny.be</a>)""",
     axelor="""Axelor (<a href="http://axelor.com">http://axelor.com</a>)""")|n}
 <br/>
-${_("Running Server:")} <span>${rpc.session.protocol}://${rpc.session.host}:${rpc.session.port} - database: ${rpc.session.db or 'N/A'}</span><br/>
             </div>
         </td>
     </tr>
