--- sidebar.mako	2010-04-07 01:57:10.000000000 -0600
+++ sidebar.mako-mod	2010-08-30 10:03:46.000000000 -0600
@@ -11,11 +11,11 @@
                                 <td>
                                     <table border="0" cellpadding="0" cellspacing="0" width="100%">
                                         <tr>
-                                            <td width="8" style="background: #ac0000"/>
-                                            <td width="7" style="background-color: #363636"/>
-                                            <td style="color: white; font-weight: bold; font-size: 12px; background-color: #363636">${_("REPORTS")}</td>
-                                            <td width="35" valign="top" style="background: url(/static/images/diagonal_left.gif) no-repeat; background-color: #666666"/>
-                                            <td width="50" style="background-color: #666666"/>
+                                            <td width="8" class="sidebarBox"/>
+                                            <td width="7" class="sidebarTextBox"/>
+                                            <td class="sidebarTextBox">${_("REPORTS")}</td>
+                                            <td width="35" valign="top" class="sidebarCorner"/>
+                                            <td width="50" class="sidebarEnd"/>
                                         </tr>
                                     </table>
                                 </td>
@@ -40,11 +40,11 @@
                                 <td>
                                     <table border="0" cellpadding="0" cellspacing="0" width="100%">
                                         <tr>
-                                            <td width="8" style="background: #ac0000"/>
-                                            <td width="7" style="background-color: #363636"/>
-                                            <td style="color: white; font-weight: bold; font-size: 12px; background-color: #363636">${_("ACTIONS")}</td>
-                                            <td width="35" valign="top" style="background: url(/static/images/diagonal_left.gif) no-repeat; background-color: #666666"/>
-                                            <td width="50" style="background-color: #666666"/>
+                                            <td width="8" class="sidebarBox""/>
+                                            <td width="7" class="sidebarTextBox"/>
+                                            <td class="sidebarTextBox">${_("ACTIONS")}</td>
+                                            <td width="35" valign="top" class="sidebarCorner"/>
+                                            <td width="50" class="sidebarEnd"/>
                                         </tr>
                                     </table>
                                 </td>
@@ -70,11 +70,11 @@
                                 <td>
                                     <table border="0" cellpadding="0" cellspacing="0" width="100%">
                                         <tr>
-                                            <td width="8" style="background: #ac0000"/>
-                                            <td width="7" style="background-color: #363636"/>
-                                            <td style="color: white; font-weight: bold; font-size: 12px; background-color: #363636">${_("LINKS")}</td>
-                                            <td width="35" valign="top" style="background: url(/static/images/diagonal_left.gif) no-repeat; background-color: #666666"/>
-                                            <td width="50" style="background-color: #666666"/>
+                                            <td width="8" class="sidebarBox"/>
+                                            <td width="7" class="sidebarTextBox"/>
+                                            <td class="sidebarTextBox">${_("LINKS")}</td>
+                                            <td width="35" valign="top" class="sidebarCorner"/>
+                                            <td width="50" class="sidebarEnd"/>
                                         </tr>
                                     </table>
                                 </td>
@@ -101,11 +101,11 @@
                                 <td>
                                     <table border="0" cellpadding="0" cellspacing="0" width="100%">
                                         <tr>
-                                            <td width="8" style="background: #ac0000"/>
-                                            <td width="7" style="background-color: #363636"/>
-                                            <td style="font: verdana; color:white; font-weight:bold; font-size:12px; background-color: #363636">${_("ATTACHMENTS")}</td>
-                                            <td width="42" valign="top" style="background: url(/static/images/diagonal_left.gif) no-repeat; background-color: #666666"/>
-                                            <td width="50" style="background-color: #666666"/>
+                                            <td width="8" class="sidebarBox"/>
+                                            <td width="7" class="sidebarTextBox"/>
+                                            <td class="sidebarTextBox">${_("ATTACHMENTS")}</td>
+                                            <td width="42" valign="top" class="sidebarCorner"/>
+                                            <td width="50" class="sidebarEnd"/>
                                         </tr>
                                     </table>
                                 </td>
